Automattic/wp-calypso
# Removing the Canva marketing card

## Proposed Changes

Our affiliate agreement with Canva is ending. pau2Xa-54c-p2
This PR does the following:
* Removes the marketing tools feature for Canva
* Removes the corresponding click event
* Removes the logo from our assets

## Testing Instructions

* Checkout branch or use the live testing link
* Navigate to **Tools** --> **Marketing**
<img width="276" alt="Screenshot 2023-07-25 at 5 25 17 PM" src="https://github.com/Automattic/wp-calypso/assets/48809176/c183a3a3-81fc-4e2a-8992-5919f145e526">

* Verify that the marketing card for Canva has been removed

**Before:**
<img width="1069" alt="Screenshot 2023-07-25 at 5 27 27 PM" src="https://github.com/Automattic/wp-calypso/assets/48809176/e7d4eb47-2866-426b-a6b8-456f1a8ab8e0">

**After:**
<img width="1090" alt="Screenshot 2023-07-25 at 5 26 13 PM" src="https://github.com/Automattic/wp-calypso/assets/48809176/87a11248-15bd-4458-a3d8-c6cac2dfcf5a">


## Pre-merge Checklist
